src/keyszer/transform.py

```diff
@@ -445,13 +445,30 @@ def transform_key(key, action, ctx):
 
         if logger.VERBOSE:
```

Comment:
This has grown so massive it really should be pulled out into a helper and passed whatever it needs:

```py
if logger.verbse:
  log_combo_context(combo, ctx, .,,)
```